.github/workflows/package.yml

```diff
@@ -1,24 +1,31 @@
 name: package
 
 on:
-#  push:
-#    branches:
-#      - next
-#      - cd_*
-#    tags:
-#      - "v[0-9]+.[0-9]+.[0-9]+"
-#  pull_request:
+  push:
+    branches:
+      - next
+      - cd_*
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
I think the workflow is only triggered when creating a tag

Reply:
@namhoang1604 , I removed branches but left workflow dispatch option. it is handy when a workflow can be ran manually.